### PR TITLE
Make the grid 2D

### PR DESCRIPTION
**Pull request by stefania-tsvetkova**
- made the grid 2 dimantional
- updated the color scheme

### CODE REVIEW

File: src/app/app.component.scss [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+  background-color: var(--dark-red);
+}
+
+.empty-cell-light {
+  background-color: var(--very-light-green);
+}
+
+.empty-cell-middle {
+  background-color: var(--light-green);
+}
```
Comment (eGreshkov) [inline src/app/app.component.scss]: Missing empty line at EoF

File: src/app/constants/game-settings.constants.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const GRID_ROWS = 15;
+
+export const GRID_COLUMNS = 20;
+
+export const STEP_TIME = 100;
```
Comment (eGreshkov) [inline src/app/constants/game-settings.constants.ts]: Add empty line

File: src/app/enums/direction.enum.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export enum Direction {
+    UP = 'UP',
+    DOWN = 'DOWN',
+    LEFT = 'LEFT',
+    RIGHT = 'RIGHT',
+};
```
Comment (eGreshkov) [inline src/app/enums/direction.enum.ts]: Add empty line

File: src/app/models/cell.model.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        }
+        
+        if (this.column < 0) {
+            this.column = GRID_COLUMNS - 1;
+        }
+        else if (this.column >= GRID_COLUMNS) {
+            this.column = 0;
+        }
+    }
+}
```
Comment (eGreshkov) [inline src/app/models/cell.model.ts]: Add empty line

File: src/app/models/snake.model.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    move() {
+        this.body = this.body.map((cell: Cell, index: number) => {
+            if (index === this.headIndex) {
+                return cell.getNeighbour(this.direction);
+            }
+            
+            return this.body[index - 1];
+        });
+    }
+}
```
Comment (eGreshkov) [inline src/app/models/snake.model.ts]: Add empty line

File: src/app/services/snake.service.ts [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        direction === Direction.LEFT || 
+        direction === Direction.RIGHT
+      ).length;
+
+      const onTheSameAxis =
+      filteredRowAxes === 2 || filteredRowAxes === 0;
+  
+      return !onTheSameAxis;
+    }
+}
```
Comment (eGreshkov) [inline src/app/services/snake.service.ts]: Add empty line

File: src/app/utilities/direction.utility.ts [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+      case 'ArrowUp':
+        return Direction.UP;
+      case 'ArrowRight':
+        return Direction.RIGHT;
+      case 'ArrowDown':
+        return Direction.DOWN;
+      default:
+        return Direction.RIGHT;
+    }
+  }
```
Comment (eGreshkov) [inline src/app/utilities/direction.utility.ts]: Add empty line

File: src/styles.scss [hunk 8]
```diff
@@ -1 +1,16 @@
...
+    // purple colors
+    --dark-purple: #7100dd;
+
+    // red colors
+    --dark-red: #ff2c41;
+}
+
+body {
+    margin: 0;
+}
```
Comment (eGreshkov) [inline src/styles.scss]: Add empty line

**Review by mgandeva — APPROVED** [review]

File: src/app/models/cell.model.ts [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+                return new Cell(this.row + 1, this.column);
+        }
+    }
+
+    equals(cell: Cell): boolean {
+        return this.row === cell.row && this.column === cell.column;
+    }
+
+    private normalize()
+    {
```
Comment (eGreshkov) [inline src/app/models/cell.model.ts]: Please move this opening brace to the previous line, to stay consistent with coding style

**Review by eGreshkov — APPROVED** [review]
LGTM, got some minor styling comments.

**Pull request merged**